uefi-test-runner/src/bin/shell_launcher.rs

```diff
@@ -0,0 +1,83 @@
+#![no_std]
+#![no_main]
+
```

Comment:
nit: maybe add 2-3 sentences high-level description about how this binary works/what purpose it fulfills?